packages/database/src/TupaiaDatabase.js

```diff
@@ -106,7 +106,6 @@ export class TupaiaDatabase {
 
   maxBindingsPerQuery = MAX_BINDINGS_PER_QUERY;
 
-  // can be replaced with 'generateTestId' by tests
   generateId = generateId;
```

Comment:
Not really a review comment, but what does this line do? 😅

Or was it just there to give a dev the opportunity to replace it with `generateId = generateTestId`?

Reply:
I think it was partly to allow this to be replaced by `generateTestId` but if I am reading it correctly I think it sets the class `generateId` method to the util `generateId` so it can be used as `this.generateId`